Doc/whatsnew/3.11.rst

```diff
@@ -906,6 +906,18 @@ sysconfig
   (Contributed by Miro Hrončok in :issue:`45413`.)
 
 
+.. _whatsnew311-tempfile:
+
+tempfile
+--------
+
+* :class:`~tempfile.SpooledTemporaryFile` now fully implements the methods
+  of the :class:`io.BufferedIOBase`/:class:`io.TextIOBase` base classes.
```

Comment:
Does the slash mean and or or?  Could the word be used to improve clarity?

Reply:
Hmm, I actually originally used `or` here, but changed it to `/` before pushing, because "or" read rather strangely to me in this context, as without adding the detail regarding which is used when (binary vs. text mode, which I didn't want to complicate this high-level summary with), it implied uncertainty in which ABC's methods were implemented as opposed to the actual meaning. `/` kept the exact relationship more unspecified, which I wasn't totally happy with but I figured a reader would want to follow the reference anyway if they wanted to know more, which explains it in suitable detail.

But I'm not sure...what do you think?

Reply:
Keep it simple but clear.

> SpooledTemporaryFile now fully implements the methods of io.BufferedIOBase or io.TextIOBase (depending on file mode).

Reply:
Implemented, thanks :+1: